scripts/scrape.py

```diff
@@ -49,4 +57,7 @@
     scraper = scraper_class(
         args.teiid_user, args.teiid_password, args.kerberos, args.neo4j_user, args.neo4j_password,
         args.neo4j_server)
-    scraper.run(since=args.since)
+    since = args.since
+    if args.days_ago:
```

Comment:
Maybe we should throw an error if the input given by the user is not an integer

Reply:
The `type=int` on line 23 should handle that already.